### PR TITLE
Fixed a couple compilation errors for MSVC 64-bit

### PR DESCRIPTION
**Pull request by marler8997**

### CODE REVIEW

File: src/util.hpp [hunk 1]
```diff
@@ -24,27 +24,12 @@
 #define ATTRIBUTE_RETURNS_NOALIAS __declspec(restrict)
 #define ATTRIBUTE_NORETURN __declspec(noreturn)
 
-static inline int clzll(unsigned long long mask) {
```
Comment (marler8997) [inline src/util.hpp]: Had to move the clzll definition lower in the file so it could see the `zig_unrechable` definition.

File: src/quadmath.hpp [hunk 2]
```diff
@@ -32,7 +32,7 @@ static inline __float128 strtoflt128(const char *s, char **sp) {
 
 static inline int quadmath_snprintf(char *s, size_t size, const char *format, ...) {
     va_list args;
-    va_start(format, args);
+    va_start(args, format);
```
Comment (andrewrk) [inline src/quadmath.hpp]: I had to switch this for MacOS so I think we need `#if defined(__APPLE__)` here.

Comment (marler8997) [reply src/quadmath.hpp]: this implementation is inside a `#if defined(_MSC_VER)`...how would it be compiled for apple?

Comment (andrewrk) [reply src/quadmath.hpp]: hmm good point. I don't know what happened that made me think we needed to switch those args. 

I'll go ahead and merge this, the fix with unreachable is good and the code I'm currently working on removes quadmath dependency anyway.

**Pull request merged**